src/display/base.jl

```diff
@@ -134,8 +134,8 @@ end
     end
     return st
   end
-  LazyTree(span(c(strong(String(nameof(typeof(d)))),
-            Atom.fade(" $(eltype(d).parameters[1]) → $(eltype(d).parameters[2]) with $(length(d)) entries"))), cs)
```

Comment:
how about changing `→`s on line 131 & 133 as well ?
It would make it more consistent with the syntax (and the `display`)

![image](https://user-images.githubusercontent.com/40514306/65373342-32bec600-dcb7-11e9-818d-10dd7dc76175.png)

Reply:
Yeah, makes sense, I suppose.